spec/factories/strategies/valkyrie_resource.rb

```diff
@@ -1,4 +1,7 @@
-class ValkyrieCreateStrategy
+# This strategy is registered in spec_helper.rb and given a name that can be called by factory bot (e.g. create_using_hyrax_adapter).
```

Comment:
i'm not sure these comments are helpful. Their truth will depend on our usage of the `FactoryBot` strategy registry throughout the application, so they'll be hard to maintain over time.

Maybe just a `@see https://github.com/thoughtbot/factory_bot/blob/master/GETTING_STARTED.md#custom-strategies` would be better?

Reply:
`create_using_hyrax_adapter` looks inaccurate.

Reply:
I started to refactor valkyrie_create to be create_using_hyrax_adapter as the companion of create_using_test_adapter, as this names are more descriptive.  But it brought in too many changes and I backed it out.  Looks like I missed this one.

I'll create a separate issue to discuss the potential change from valkyrie_create to create_using_hyrax_adapter.

Reply:
thanks!

Reply:
DONE: Removed comments referencing spec_helper.rb and added @see to factory bot documentation